File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Entity/Commons/ContactPlaceName.cs
using System;
using System.Collections.Generic;
using VecompSoftware.DocSuiteWeb.Entity.Protocols;

namespace VecompSoftware.DocSuiteWeb.Entity.Commons
{
    public class ContactPlaceName : DSWBaseEntity
    {
        #region [ Constructor ]
        public ContactPlaceName() : this(Guid.NewGuid()) { }
        public ContactPlaceName(Guid uniqueId)
            : base(uniqueId)
        {
            Contacts = new HashSet<Contact>();
            ProtocolContactManuals = new HashSet<ProtocolContactManual>();
        }

        #endregion

        #region[ Properties ]

        public string Description { get; set; }

        #endregion

        #region [ Navigation Properties ]
        public virtual ICollection<Contact> Contacts { get; set; }
        public virtual ICollection<ProtocolContactManual> ProtocolContactManuals { get; set; }
        #endregion
    }
}

File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Data/Domain/Common/Contact/ContactPlaceName.vb
<Serializable()> _
Public Class ContactPlaceName
    Inherits DomainObject(Of Int32)


#Region "private data"
    Private _description As String
#End Region

#Region "Properties"
    Public Overridable Property Description() As String
        Get
            Return _description
        End Get
        Set(ByVal value As String)
            _description = value
        End Set
    End Property

#End Region

#Region " Constructor "
    Public Sub New()
        _description = ""
        UniqueId = Guid.NewGuid()
    End Sub
#End Region

End Class

